docs/self-hosted/oel/oauth2/changelog.mdx

```diff
@@ -0,0 +1,13 @@
+---
+id: changelog
+title: Changelog for Ory Hydra OEL
+sidebar_label: Changelog
+---
+
+## 2024-07-17
+
+### **feat: disable pre-configured multi-region config in Ory Hydra ([#6638](https://github.com/ory-corp/cloud/issues/6638))**
```

Comment:
side not : should this be here? link points to private repo so either way won't provide context for anyone outside.

Reply:
No, but I could not find an easy way to remove it in chglog generator :(

Reply:
We have improved changelog generation so it does not have internal link now, please see my suggestion

Reply:
We will add this changelog to upstream repo so that it is consistent with next CI runs. We can merge is as it now here, upstream repo CI will update the file without internal links.